src/services/teams/hooks/index.js

```diff
@@ -515,9 +518,32 @@ const testChangesForPermissionRouting = globalHooks.ifNotLocal(async (hook) => {
 			throw new Forbidden('Can not adding users from other schools.');
 		}
 		if (isLeaveTeam) {
-			wait.push(leaveTeam(hook).catch((err) => {
-				throw new Forbidden('Permission LEAVE_TEAM is missing.');
-			}));
+			let amountOfTeamowner = 0; // only counter, do not have to be the real amount of Teamowners!
+			let isTeamowner = false;
+			for (const user of team.userIds) {
+				if (user.userId.toString() === sessionUserId) {
+					if (hook.findRole('_id', user.role, 'name') === 'teamowner') {
```

Comment:
however shouldnt this be hook.hasRole instead?

Reply:
I could rember that it is a spezial function written by cedric for this case